tailwind.config.js

```diff
@@ -1,10 +1,36 @@
 module.exports = {
   content: ['./src/**/*.{js,jsx,ts,tsx}'],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        primary: {
+          dark: '#3dc094',
+          light: '#309976',
+        },
+        secondary: {
+          dark: '#373445',
+          light: '#e9e0ff',
+        },
+        'brand-text': {
+          dark: '#f0f2f5',
+          light: '#373445',
+        },
+        background: {
+          normal: {
+            dark: '#373445',
+            light: '#ffffff',
+          },
+          darker: {
+            dark: '#32303f',
+            light: '#f2f2f2',
+          },
+        },
+      },
+    },
   },
   corePlugins: {
     preflight: false,
   },
   plugins: [],
+  darkMode: ['selector', '[data-theme="dark"]'],
```

Comment:
Should we not try to use the default `prefers-color-scheme` to make it more user friendly?

Reply:
Will merge this PR and leave this comment for your consideration.

Reply:
The theme detection is done by Docusaurus, it will add the attribute `data-theme="light | dark"` to the `html` tag. Tailwind here relies on the attribute for the `dark:*` classes.
I think your suggestion is good, if we want to do this, we will have to configure it through Docusaurus, by setting `respectPrefersColorScheme` [(ref)](https://docusaurus.io/docs/2.x/api/themes/configuration#respectPrefersColorScheme).
Let's add this point in the issue and we will work on it in the future.